src/appShell/App/PortalFooter.tsx

```diff
@@ -33,11 +27,18 @@ export default class PortalFooter extends React.Component<{}, IPortalFooterState
     }
 
     render() {
+            var version;
+            if (this.props.appStore.portalVersion.isComplete && !this.props.appStore.portalVersion.isError && getBrowserWindow().globalStores.appStore.portalVersion.result){
```

Comment:
@dippindots one more thing i just noticed: you're still using the global reference to appStore in last item here.

Reply:
Sorry about that, I just fixed it.